### ISSUE TITLE
`--version` flag is not being forwarded to nx builder

### ISSUE THREAD

**Issue by gibsonjoshua55** [jscutlery/semver#160]
## Current Behavior
When using the nx cli and passing `--version=prerelease` that option never makes it into my builder. It's as if I never passed the option

```
npx nx run workspace:version --version=prerelease --preid=beta --dry-run

> nx run workspace:version --preid=beta --dry-run --dryRun
```

## Expected Behavior
passing in `--version=x` would actually get recognized as an option by the builder

## Nx Issue?
I'm not quite sure if this is an issue with semver or with nx. I'm curious because it seems like an issue with nx from what I've dug into (as the option never even reaches the runBuilder function in runtime), but I can't believe that I'm the only user who is facing this issue.  I have opened an issue in the nrwl/nx repo and will see if they have thoughts, but I'm curious if anyone else here has run into or seen this issue.

### Environment
```

  Node : 12.22.1
  OS   : darwin x64
  yarn : 1.22.10
  
  nx : Not Found
  @nrwl/angular : 12.0.8
  @nrwl/cli : 12.0.8
  @nrwl/cypress : 12.0.8
  @nrwl/devkit : 12.0.8
  @nrwl/eslint-plugin-nx : 12.0.8
  @nrwl/express : Not Found
  @nrwl/jest : 12.0.8
  @nrwl/linter : 12.0.8
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 12.0.8
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.2.12
  @nrwl/web : Not Found
  @nrwl/workspace : 12.0.8
  @nrwl/storybook : Not Found
  @nrwl/gatsby : Not Found
  typescript : 4.1.5
  ```

**Comment by yjaaidi** [comment]
Hi @gibsonjoshua55! It is an Nx issue as you discovered by yourself https://github.com/nrwl/nx/issues/5432 😊
We are waiting for the next Nx release (>11.6.3) to solve this.

**Comment by edbzn** [comment]
Should be fixed in 2.8.1.
